hail/python/hail/utils/hadoop_utils.py

```diff
@@ -84,10 +101,22 @@ def hadoop_copy(src, dest):
     """Copy a file through the Hadoop filesystem API.
     Supports distributed file systems like hdfs, gs, and s3.
 
+    Hint
```

Comment:
hint is an invalid directive - change to Note? and move below the examples?

Reply:
What am I missing, why don't all admonitions listed by Read The Docs work? https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#admonitions